MAINT: Bump to 0.11dev0

larsoner opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Sibling PR to https://github.com/conda-forge/mne-feedstock/pull/107. I want to test a 1.2 MNE installer before actualy release, this should help because I can build a `dev` installer ahead of time and debug any issues.

larsoner committed:
MAINT: Bump to 0.11dev0

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

larsoner commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-build 3.22.0, conda-smithy 3.21.2, and conda-forge-pinning 2022.10.06.21.48.30

larsoner committed:
FIX: Reqs

larsoner committed:
FIX: Add self

larsoner commented:
Okay I synced with `main` using `git diff upstream/main recipes/meta.yml`, added myself as a maintainer (hope that's okay!), then pushed. Feel free to merge when happy. It should come back green because I added the `importlib_resources` req here... though I still think we should get rid of it in https://github.com/mne-tools/mne-bids/pull/1076

hoechenberger commented:
Thanks @larsoner
